DeAceroDLabs/marketplace-frontend
# [DMP-549] Product Detail URL

## What is done

[[DMP-549] URL Detalle de Producto](https://deacero.atlassian.net/browse/DMP-549)

## How is done

1. Added a new route `producto/:productId`.
2. Created a new `ProductDetail` view component.
3. Made the product cards clickable

## How to test

1. Run the project via `npm start`
2. Click on a product card on the home screen or the search screen.
4. You should see the title of the product matching the product id.
